### PR TITLE
Replace default ECS task definition AmazonEC2ContainerServiceFullAccess policy to AmazonECSFullAccess policy

### LINKED ISSUE #610: Replace default AmazonEC2ContainerServiceFullAccess to AmazonECS_FullAccess
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/managed-policy-transition.html
Amazon is going to deprecate AmazonEC2ContainerServiceFullAccess policy

https://github.com/pulumi/pulumi-awsx/blob/2506065d78d21f88fea54d28f240c8e27d2fae3c/nodejs/awsx/ecs/taskDefinition.ts#L152

### PR DESCRIPTION
**Pull request by ahanoff**
Close #610

### CODE REVIEW

File: nodejs/awsx/package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
         "mime": "^2.0.0"
     },
     "peerDependencies": {
-        "@pulumi/aws": "^1.0.0 || ^2.0.0 || ^3.0.0",
+        "@pulumi/aws": "^3.22.0",
```
Comment (ahanoff) [inline nodejs/awsx/package.json:22]: https://github.com/pulumi/pulumi-awsx/issues/610#issuecomment-750413184

**Comment by leezen** [comment]
Oops. I realized I opened https://github.com/pulumi/pulumi-awsx/pull/624 and forgot that this PR was here. I think if you remove the Lambda change (since we actually don't want to make that swap and plan to do something different for that), we should be good to go here.

**Comment by ahanoff** [comment]
> Oops. I realized I opened #624 and forgot that this PR was here. I think if you remove the Lambda change (since we actually don't want to make that swap and plan to do something different for that), we should be good to go here.

hi @leezen  I'll update tonight. Thanks!

**Comment by leezen** [comment]
@ahanoff Sorry, but I went ahead and merged the other PR in since the deprecation is happening quite soon and wanted to get this change out there.

**Comment by ahanoff** [comment]
> @ahanoff Sorry, but I went ahead and merged the other PR in since the deprecation is happening quite soon and wanted to get this change out there.

no issue @leezen , thanks 👍

**Pull request closed without merging**